Source/stores.h

```diff
@@ -23,7 +23,7 @@ namespace devilution {
 #define STORE_LINES 104
 
 enum class TalkID : uint8_t {
-	None,
+	Exit,

```

Comment:
This seems an odd renaming

Reply:
I was about to say the same thing. When I look at the other values in that enum, `None` makes a lot more sense to me than `Exit`. Maybe something like `void ExitStore() { ActiveStore = TalkID::None; }` would serve the same purpose, but less awkward.

Reply:
It makes more sense in the linked PR, but I removed it here. `ExitStore()` is a function that does exist in the linked PR, but named `IsPlayerInStore()` as a bool function.